extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -362,14 +372,20 @@ export class ConsentUI {
         data['initialHeight'].indexOf('vh') >= 0
       ) {
         const dataHeight = parseInt(data['initialHeight'], 10);
+        // Set initialHeight to max height fallback if applicable
+        this.initialHeight_ =
+          dataHeight >= 80 ? MAX_INITIAL_HEIGHT : this.initialHeight_;
 
-        if (dataHeight >= 10 && dataHeight <= 60) {
+        if (dataHeight >= 10 && dataHeight <= 80) {
           this.initialHeight_ = `${dataHeight}vh`;
+          this.lightboxEnabled_ = dataHeight > 60;
```

Comment:
Again, why `60`? Create a variable.

Reply:
60 was the original max allowed VH for the bottom sheet. So anything over that will trigger modal view.